src/app/_services/index.ts

```diff
@@ -6,3 +6,5 @@ export { VolumeService } from './volume/volume.service';
 export { MockService } from './mock/mock.service';
 export { TaskService } from './task/task.service';
 export { ConfigurationService } from './configuration/configuration.service';
+// export { ServicesService } from './services/services.service';
+// Cannot add above because it causes a circular dependency due to configurationService being used in ServicesService
```

Comment:
Had a look at how this problem was over come in Volumes/Networks and will fix it now.